desktop-exporter/app/components/traceList.tsx

```diff
@@ -40,11 +72,11 @@ export function TraceList(props: TraceListProps) {
         height={size ? size.height : 0}
         itemData={props.traceSummaries}
         itemCount={props.traceSummaries.length}
-        itemSize={50}
+        itemSize={110}
```

Comment:
Since you're using `height` and `marginTop` here to calculate this, it might be a good idea to factor those out into constants in this file. That way changing them in the future is simpler and less fussy.

```js
const listItemHeight = 100;
const listItemTopMargin = 10;
```

Reply:
Good call, since these numbers will change once we iron out the info we want in TraceSummary, and what we want to display in this list.